boards/arm/beagleconnect_freedom_cc1352r/Kconfig.defconfig

```diff
@@ -0,0 +1,12 @@
+# BeagleBoard BeagleConnect Freedom
```

Comment:
Compare to https://github.com/jadonk/beagle_connect_zephyr_board/blob/93a5a3f96b0c8b47634218c9d7a40741459c316d/Kconfig.defconfig and credit Erik Larson (@statropy). Again, not CC1352R.